packages/esm-patient-chart-app/src/visit/visit-action-items/delete-visit-action-item.component.tsx

```diff
@@ -40,7 +40,13 @@ const DeleteVisitActionItem: React.FC<DeleteVisitActionItemProps> = ({ patientUu
   return (
     <UserHasAccess privilege="Delete Visits">
       {compact ? (
-        <IconButton onClick={deleteVisit} label={getCoreTranslation('delete')} kind="ghost" size={responsiveSize}>
+        <IconButton
+          onClick={deleteVisit}
+          label={getCoreTranslation('delete')}
+          kind="ghost"
+          size={responsiveSize}
+          align="top-end"
```

Comment:
Added the `align` property to some buttons so that they no longer cause the table to become horizontally scrollable when hovered.